### DISCUSSION TITLE
lack of documention for Phalcon 5.  sometimes literally ZERO documention

### DISCUSSION THREAD

**Discussion by zhangster** [phalcon/cphalcon#16386] [General]
i find this really frustrating.  i'm upgrading from Phaclon 3.4 to 5.0.    That broke a lot of things, including many of the HTML helps.  The checkbox function no longer works, and there's NO documention whatsoever, anywhere, and how to to use the new checkbox helper.  I don't know what the function expects, whatever the escaper function is.      

There's also no examples of checkboxes in their INVO application.

If Phalcon is expecting people to use their framework, they need to at least have all the basics covered.

**Comment by hakimio** [comment]
Have you read the upgrade guides? Specifically [this section](https://docs.phalcon.io/5.0/en/upgrade#forms:~:text=The%20classes%20now,the%20checked%20parameter.).

**Comment by kgrammer** [comment]
I migrated four sites from Phalcon 3, to 4 and then now to 5.  In all three cases, I followed the various upgrade docs (3 to 4, and the 4 to 5) and found them to be very informative and easy to follow.

I can't speak to the checkbox issue as I use VOLT for building and managing my HTML content.  However, a quick search of the V5 docs provided this link: https://docs.phalcon.io/latest/html-tagfactory/?h=checkbox#inputcheckbox

It looks like it's describing the input parameters and provides sample code.
